src/build-runtime.cmake

```diff
@@ -111,7 +111,7 @@ add_custom_command(
 add_executable(runtime ${CMAKE_CURRENT_BINARY_DIR}/runtime.4.o notify.c)
 # CMake gets confused by the .o object, therefore we need to tell it that it shall link everything using the C compiler
 set_property(TARGET runtime PROPERTY LINKER_LANGUAGE C)
-target_link_libraries(runtime PRIVATE libsquashfuse dl xz libzlib pthread libappimage_shared libappimage_hashlib)
+target_link_libraries(runtime PRIVATE libsquashfuse fuseprivate squashfuse_ll dl xz libzlib pthread libappimage_shared libappimage_hashlib fuse lzo2)
```

Comment:
`libsquashfuse` is an imported library; `fuseprivate` and `squashfuse_ll` mean nothing to CMake and are just passed to the linker. This might work, but it's unsafe. It's the wrong place for this.

`fuse` and `lzo2` have never been imported into CMake either. We don't even *want* to link to fuse directly.

None of these libraries should have to be listed here. They need to be listed as dependencies of `libsquashfuse`, if any.